packages/authentication/package.json

```diff
@@ -4,9 +4,10 @@
   "description": "A LoopBack component for authentication support.",
   "scripts": {
     "acceptance": "mocha --opts ../../test/mocha.opts 'test/acceptance/**/*.ts'",
-    "build": "npm run build:lib && npm run build:lib6",
+    "build": "npm run build:lib && npm run build:lib6 && npm run build:apidocs",
```

Comment:
... And BTW I feel it makes more sense to bundle the api docs together because they represent the doc generated from the code state when it was published. On-the-fly generation can give different state it the sdocs tool has changed. Although it will be latest, but it will be confusing to the users. **Docs should be tightly coupled with code state**, not sdocs tool state